corehq/apps/reports/util.py

```diff
@@ -84,7 +84,7 @@ def _create_temp_user(user_id):
             return temp_user
         return None
 
-    user_ids = user_ids if user_ids and user_ids[0] else None
+    user_ids = user_ids if bool(user_ids) else None
```

Comment:
I think this changes the behavior if the list contains one empty value - which is explicit enough that it seems like it might have been intentional.

```python
>>> user_ids = ['']
>>> user_ids and user_ids[0] 
''   # will evaluate to False as a boolean
>>> bool(user_ids)
True
```

looking below here it seems like this is just used in a `if user_ids is not None` check later, which seems kind of unnecessary. it might be easier to write this as:
```python
user_ids = user_ids or []  # make sure it's iterable
user_ids = filter(None, user_ids)  # remove empty values
```
and then change the check to just be `if user_ids` instead of `if user_ids is not None`